go/vt/mysqlctl/mysqld.go

```diff
@@ -286,7 +293,7 @@ func (mysqld *Mysqld) RunMysqlUpgrade() error {
 			return fmt.Errorf("can't dial mysqlctld: %v", err)
 		}
 		defer client.Close()
-		return client.RunMysqlUpgrade(context.TODO())
+		return client.RunMysqlUpgrade(ctx)
 	}
```

Comment:
Seems simplest to pass in the context we already have in all the callers further up in the call chain.